RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chrome.java

```diff
@@ -199,11 +200,12 @@ private void getBookmark() {
             if (bookmarkFile.getSize() == 0) {
                 continue;
             }
-            String temps = RAImageIngestModule.getRATempPath(currentCase, "chrome") + File.separator + bookmarkFile.getName().toString() + j + ".db"; //NON-NLS
+            String temps = RAImageIngestModule.getRATempPath(currentCase, "chrome") + File.separator + bookmarkFile.getName() + j + ".db"; //NON-NLS
             try {
                 ContentUtils.writeToFile(bookmarkFile, new File(temps), context::dataSourceIngestIsCancelled);
             } catch (IOException ex) {
-                logger.log(Level.SEVERE, "Error writing temp sqlite db for Chrome bookmark artifacts.{0}", ex); //NON-NLS
+                logger.log(Level.WARNING, String.format("Error writing temp sqlite db file '%s' (id=%d) for Chrome bookmark artifacts.",
+                        bookmarkFile.getName(), bookmarkFile.getId()), ex); //NON-NLS
```

Comment:
Does the addErrorMessage call following this mean that we are going to get double logging? If so, this is a consistent pattern in this PR.